### ISSUE TITLE
Commit summary text not accept letters as input that are shortcuts in the menu

### ISSUE THREAD

**Issue by MikeWise2718** [github-for-unity/Unity#294]
### Description

I have a few shortcuts enabled for my unity program using the menu (the _s below):

### Steps to Reproduce

1. Define a Menu Item in your program using a shortcut (like _s)
2. Try and check the change into the Github for Unity window
3. Notice that it no longer accepts the that letter (lowercase s). Capitals still work.

Here is the code that sets it the menu item with the shortcut.

        [MenuItem("--LinkCloud/Split Selected Link _s")]
        static void SplitlSelLink()
        {
            var selobj = UnityEditor.Selection.activeGameObject;
            if (selobj != null)
            {
                rman.SplitLink(selobj.name);
            }
            //  rman.DeleteSelectedLink();
        }

I am using Unity 2017.1.1f1 Personal for this.

[github-unity.log.txt](https://github.com/github-for-unity/Unity/files/1289561/github-unity.log.txt)

![menuscreenshot](https://user-images.githubusercontent.com/7562887/30237255-5e6ba782-952e-11e7-9ca3-b76183a5d2f6.png)

![commitwindowscreenshot](https://user-images.githubusercontent.com/7562887/30237283-ef7e832a-952e-11e7-9870-501d6153f8ea.png)

**Comment by shana** [comment]
Is it also broken in other text fields, or is it only specifically failing in the commit summary?

**Comment by shana** [comment]
I've tested adding a menu item with "s" as the activator and then typing in the search field in the hierarchy, and the menu also grabs input before it reaches the text field, so this is a Unity bug. I've tested 5.4, 5.5, 5.6 and 2017.1 and they all behave the same, so this has been broken for a while.  Unity is already tracking this bug in https://issuetracker.unity3d.com/issues/custom-menuitem-hotkey-takes-priority-over-input-field so hopefully they'll get around to fixing it soon. In the meantime, avoiding single key accelerators is probably best :-/
